src/Schema/Values/OwnerType.ts

```diff
@@ -73,7 +75,9 @@ export type ScreenOwnerType =
   | OwnerType.fair
   | OwnerType.fairArtworks
   | OwnerType.home
-  | OwnerType.inbox
```

Comment:
I think we should fire different screen events with a context and owner for the 'Inquiries' view and the 'Bids' view within inbox. This is a unique pattern AFAIK but allows us to group all screen views as type 'inbox' at the session level while still being able to distinguish between views if need be.